addinstanceform.php

```diff
@@ -39,7 +39,9 @@ class cachestore_redis_addinstance_form extends cachestore_addinstance_form {
     protected function configuration_definition() {
         $form = $this->_form;
 
-        $form->addElement('text', 'server', get_string('server', 'cachestore_redis'), array('size' => 24));
+        $form->addElement('checkbox', 'clustermode', 'Cluster Mode');
```

Comment:
"Redis introduces cluster support as of version 3.0.0"  What if we have the redis library which doesn't support cluster?  Should we check it here and give relevant form?